frameworks/digital-outcomes-and-specialists/manifests/edit_brief.yml

```diff
@@ -57,128 +57,102 @@
+      - a proposal explaining how they plan to meet your requirements
 
-    To find the most suitable supplier, you must evaluate your shortlist on:
+    You must tell suppliers how you’ll evaluate and assess them.
 
-    - technical competence, using essential skills and experience, nice-to-have skills and experience, and their proposal
-    - ‘cultural fit’, eg how the supplier will work in your organisation
-    - the price of the proposal
   questions:
+    - numberOfSuppliersOutcomes
     - weightingOutcomes
-    - essentialRequirementsOutcomes
-    - niceToHaveRequirementsOutcomes
-    - successCriteriaOutcomes
+    - technicalCompetenceCriteriaOutcomes
     - culturalFitCriteriaOutcomes
     - priceCriteria
     - evaluationTypeOutcomes
 
 -
-  name: Evaluation criteria
+  name: Shortlist and evaluation criteria
   editable: True
   step: 2
   description: |
-    You need to tell suppliers how you’re going to evaluate them when you publish your requirements.
+    There are 3 stages to finding the specialist that best meets your needs.
+
+    **1\. View specialists who applied**<br />
+    When specialists apply, they declare which of your ‘essential’ and ‘nice&#8209;to&#8209;have’ skills and experience they have. Specialists who don’t meet your essential skills and experience are excluded.
+
+    **2\. Create a shortlist**<br />
+    Once you have a list of eligible specialists, you can exclude the specialists who:
+
+    - have the least ‘nice&#8209;to&#8209;have’ skills and experience
+    - can’t start when you need them to
+
+    You may need to ask for evidence of skills and experience if you have too many specialists.
+
 
-    To find the most suitable supplier, you must evaluate your shortlist on:
+    **3\. Evaluate your shortlist**<br />
+    Ask the shortlisted specialists for evidence of their skills and experience.
+
+    You must tell specialists how you’ll evaluate and assess them.
 
-    - technical competence, using the specialists’ essential skills and experience, and nice-to-have skills and experience
-    - ‘cultural fit’, eg how the supplier will work in your organisation
   questions:
+    - numberOfSuppliersSpecialists
     - weightingSpecialists
-    - essentialRequirementsSpecialists
-    - niceToHaveRequirementsSpecialists
+    - technicalCompetenceCriteriaSpecialists
     - culturalFitCriteriaSpecialists
     - evaluationTypeSpecialists
 
 -
-  name: Evaluation criteria
+  name: Shortlist and evaluation criteria
   editable: True
   step: 2
   description: |
-    You need to tell suppliers how you’re going to evaluate them when you publish your requirements.
+    There are 3 stages to finding the supplier that best meets your needs.
+
+    **1\. View suppliers who applied**<br />
+    When suppliers apply, they declare which of your ‘essential’ and ‘nice&#8209;to&#8209;have’ skills and experience they have. Suppliers who don’t meet your essential skills and experience are excluded.
```

Comment:
Quotes for `essential` and `nice-to-have` in the first sentence but not the second?